easybuild/easyconfigs/n/netcdf4-python/netcdf4-python-1.2.6-intel-2016a-Python-3.5.1.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'PythonPackage'
```

Comment:
No, I didn't know.  Probably what happened was that I just copied the existing 1.2.2 easyconfig, and saw that there was no easyblock statement, and put that in there. Any reason why there is a specific easyblock? It seems to work just fine with my config.

In case you're wondering, I'm not depending on numpy module since numpy is included in the Python one.